### PR TITLE
[Sock] event in recv thread

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Update code to handle sock event in recv thread.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: src/libnnstreamer-edge/nnstreamer-edge-internal.c [hunk 1]
```diff
@@ -715,64 +713,75 @@ _nns_edge_message_handler (void *thread_data)
...
+    poll_fd.revents = 0;
 
-    if (cmd.info.cmd == _NNS_EDGE_CMD_ERROR) {
-      nns_edge_loge ("Received error, stop msg thread.");
-      _nns_edge_cmd_clear (&cmd);
-      remove_connection = true;
-      break;
-    }
+    /* 10 milliseconds */
+    if (poll (&poll_fd, 1, 10) > 0) {
```
Comment (gichan-jang) [inline src/libnnstreamer-edge/nnstreamer-edge-internal.c:729]: Don't need to handle error case except timeout?

Comment (jaeyun-jung) [reply src/libnnstreamer-edge/nnstreamer-edge-internal.c:729]: waiting for new data here, timeout is no data received.

Comment (jaeyun-jung) [reply src/libnnstreamer-edge/nnstreamer-edge-internal.c:729]: And, _nns_edge_cmd_receive() checks connection state internally.

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**